.env.sample

```diff
@@ -0,0 +1,8 @@
+DATABASE_URL = postgres://database_user:password@host:5432/database_name
```

Comment:
This should be `DATABASE_URL=postgres://database_user:password@host:5432/database_name`

Reply:
Thank you, I am on it.